editoast/src/main.rs

```diff
@@ -118,6 +124,56 @@ fn init_sentry(args: &RunserverArgs) -> Option<ClientInitGuard> {
     }
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+struct Claims {
+    sub: String,
+    exp: usize,
+}
+
+#[derive(Debug, Deserialize)]
+pub struct Jwk {
+    kty: String,
+    #[serde(rename = "use")]
+    jwt_use: String,
+    n: String,
+    e: String,
+    kid: String,
+    alg: Option<String>,
+}
+
+#[derive(Debug, Deserialize)]
+struct JwksResponse {
+    keys: Vec<Jwk>,
+}
+
+async fn get_jwks(domain: &str) -> Result<Vec<Jwk>, reqwest::Error> {
+    let jwks_uri = format!("{}.well-known/jwks.json", domain);
+    let jwks: JwksResponse = reqwest::get(&jwks_uri).await?.json().await?;
+    Ok(jwks.keys)
+}
```

Comment:
We can improve this function by:
- Having a list of `jwks_uri` as the argument
- Don't use a domain but full URI (What you did is specific to auth0)
- This function should return a `jsonwebtoken::jwk::JwkSet`.
- To do so, you can deserialize multiple JwkSet and then merge them into a single one.